src/pages/ABTesting.vue

```diff
@@ -18,21 +18,13 @@ export default {
   },
 
   created() {
-    if (!this.isLoggedIn) {
-      this.$router.push({
-        name: 'PhoneSignIn',
-        params: { id: this.id, type: 'experiment'}
-      });
-    } else {
-      // get variant and redirect user
-      this.getAssignment();
-    }
+    this.getAssignment();
   },
-  computed: mapState(['isLoggedIn', 'userId']),
+  computed: mapState('auth', ['userId']),
```

Comment:
I'm not completely aware of the functionality here so I thought to avoid making much changes here. Do you think we should target these cleanup changes separately? As I see this PR is going more towards cleanup now

Reply:
tracking this here: #156